src/app/components/InputForm/components/SurveyField/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { bool, objectOf, any, string, shape } from 'prop-types';
+
+import { ALTERNATIVE_INPUT_TYPES } from './constants';
+import styles from './styles.module.scss';
+
+const SurveyField = ({ input, type, meta: { touched, error, warning }, className, placeholder }) => (
+  <div className={styles.inputForm}>
+    {type === ALTERNATIVE_INPUT_TYPES.textarea.type ? (
+      <textarea
+        {...input}
+        cols={ALTERNATIVE_INPUT_TYPES.textarea.cols}
+        rows={ALTERNATIVE_INPUT_TYPES.textarea.rows}
+        className={`${className} ${touched && (error || warning) ? styles.inputFormError : ''}`}
```

Comment:
```suggestion
        className={`${className} ${touched && (error || warning) && styles.inputFormError}`}
```
`''` Is not necessary